src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -584,22 +584,35 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
     });
   };
 
-  zoomOnCurrentAsset = async (currentAsset: Element | null) => {
-    if (!currentAsset || !this.pinchZoomInstance) {
+  handleSearchResult = (foundElement: Element | null) => {
+    this.zoomOnElement(
+      foundElement,
+      this.props.searchZoom || this.props.initialZoom
+    );
+  };
+
+  /**
+   * @param element - any annotation element on SVG, typically asset or document
+   * @param zoomLevel
+   */
+  zoomOnElement = (element: Element | null, zoomLevel?: number) => {
+    if (!element || !this.pinchZoomInstance) {
       return;
     }
-    const defaultZoom = this.state.isDesktop ? zoomLevel * 5 : zoomLevel * 10;
-    this.pinchZoomInstance.zoomFactor = this.props.initialZoom || defaultZoom;
+    const defaultZoom = this.state.isDesktop
```

Comment:
I'd rename this const to `zoomLevelDefault` so that it's clearer that it's a backup for the `zoomLevel` param

Reply:
makes sense but when I did it the code started to look like 

```
    const zoomLevelDefault = this.state.isDesktop
      ? defaultZoomLevel * 5
      : defaultZoomLevel * 10;
```
zoomLevelDefault <-> defaultZoomLevel looks creepy so changed constant names too